CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Before v3.0.0  |  v3.0.0
 
 - **Change:** Support Android 7 - API 24 as minimum version.
 - **Change:** Convert `UserInfoBridgeDispatcher` into interface. Usages in your code of `object : UserInfoBridgeDispatcher()` should be changed to `object : UserInfoBridgeDispatcher`.
+- **Change:** Maven Group ID changed to `io.github.rakutentech.miniapp`. You must update your dependency declaration to `io.github.rakutentech.miniapp:miniapp:3.0.0`.
+- **Change:** Migrated publishing to Maven Central due to Bintray/JCenter being [shutdown](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/). You must add `mavenCentral()` to your `repositories`.
```

Comment:
Is there any change on userguide for miniapp sdk config?

Reply:
Yeah, I updated the dependency declaration. 

I also added a [note](https://github.com/rakutentech/android-miniapp/pull/270/files#diff-e0502b4de6bd41029198132f0b09748d40b52a794167c81874134ff7a8021447R705). This note is the thing I'm a little concerned about. If the Host App is using another SDK which uses Manifest Config or SDK Utils with the old dependency declaration of `com.rakuten.tech.mobile`, then they will need to add an exclude for that dependency. 

Do you think this is okay?

Reply:
Yeah the updated userguide looks good.